packages/matrix-identity-server/src/index.test.ts

```diff
@@ -1250,6 +1264,38 @@ describe('Use configuration file', () => {
         expect(response.body).toHaveProperty('token')
         expect(response.body.token).toMatch(/^[a-zA-Z0-9]{64}$/)
       })
+      it('should accept a valid phone number request', async () => {
+        const mockResponse = Promise.resolve({
+          ok: false,
+          status: 400,
+          json: () => {
+            return {
+              errcode: 'M_INVALID_PEPPER',
+              error: 'Unknown or invalid pepper - has it been rotated?'
+            }
+          }
+        })
+        // @ts-expect-error mock is unknown
+        fetch.mockImplementation(async () => await mockResponse)
+        await mockResponse
+        const response = await request(app)
+          .post('/_matrix/identity/v2/store-invite')
+          .set('Authorization', `Bearer ${validToken}`)
+          .set('Accept', 'application/json')
+          .send({
+            address: '+33612345678',
```

Comment:
"address" looks a bad name here

Reply:
The name has to be the same for the different media. But the spec only supports mail so this is named address for the moment.

Reply:
Looking at others API, `phone` is used to designate phone when type is "pĥone"

Reply:
I think the most common use is "country" (The two-letter uppercase ISO-3166-1 alpha-2 country code that the number in phone_number should be parsed as if it were dialled from) + "phone_number"

Reply:
No, there is an encoding convention for phone in Matrix spec: international number without `+` or spaces